packages/create-sitecore-jss/src/templates/nextjs/sitecore/config/{{appName}}.config

```diff
@@ -145,19 +145,21 @@
     -->
     <layoutService>
       <configurations>
-        <config name="default">
-          <rendering>
-            <renderingContentsResolver>
+        <config name="next-default">
+          <rendering ref="/sitecore/layoutService/configurations/config[@name='default']/rendering">
+            <renderingContentsResolver ref="/sitecore/layoutService/configurations/config[@name='default']/rendering/renderingContentsResolver">
               <IncludeServerUrlInMediaUrls>false</IncludeServerUrlInMediaUrls>
             </renderingContentsResolver>
           </rendering>
+          <serialization ref="/sitecore/layoutService/configurations/config[@name='default']/serialization"/>
         </config>
-        <config name="jss">
-          <rendering>
-            <renderingContentsResolver>
+        <config name="next-jss">
```

Comment:
I'm not sure this "jss" config is even used any longer for Next.js. I think we could actually remove it (would need to verify).